packages/api/models/files.js

```diff
@@ -24,11 +46,15 @@ class FileClass extends Manager {
    */
   createFilesFromGranule(granule) {
     const fileRecords = (granule.files || [])
-      .map((file) => ({
-        granuleId: granule.granuleId,
-        bucket: file.bucket,
-        key: file.key
-      }));
+      .map((file) => {
```

Comment:
So it definitely happens and what's indexed is dependent on the output of the workflow. What would happen is it would fail and then things would not get indexed properly. This handles both cases so whatever comes out of the workflow can be handled.